place_receivers/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 3.9)
+
+project(place_receivers LANGUAGES C CXX)
+
+set(LOG_LEVEL "info" CACHE STRING "Log level for the code")
+set_property(CACHE LOG_LEVEL PROPERTY STRINGS "debug" "info" "warning" "error")
+
+set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
```

Comment:
`cmake` directory is not checked in, but will be needed for `FindXYZ`.

Reply:
ok there is no FindXYZ needed for this project.